libs/stokesian_dynamics/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# temporarily disabled to at least be able to test the CPU version
+# if(STOKESIAN_DYNAMICS_GPU)
+#   cuda_add_library(sd_gpu SHARED sd_gpu.cu)
+#   target_link_libraries(sd_gpu PRIVATE
+#     ${CUDA_CUBLAS_LIBRARIES}
+#     ${CUDA_cusolver_LIBRARY})
+# endif()
+
+if(STOKESIAN_DYNAMICS)
+  add_library(sd_cpu SHARED sd_cpu.cpp)
+  target_link_libraries(sd_cpu 
+    PRIVATE
+      ${BLAS_LIBRARIES}
+      ${LAPACK_LIBRARIES}
+    PUBLIC
+      Random123)
```

Comment:
does the installation error in CI go away if you add this?
```cmake
install(TARGETS sd_cpu LIBRARY DESTINATION ${PYTHON_INSTDIR}/espressomd)
```

Reply:
same fix for the `sd_gpu` library, can be tested locally with `make check_cmake_install`

Reply:
actually it might be better to add these `install()` commands not in this file (it's a header library) but in `src/core/stokesian_dynamics/CMakeLists.txt` inside the `if (STOKESIAN_DYNAMICS*)` blocks, because it's where we compile and install libraries.